lib/stub_ui/lib/src/ui/painting.dart

```diff
@@ -1076,6 +1076,37 @@ class Paint {
     _paintData.shader = value;
   }
 
+
+  /// The [ImageFilter] to use when drawing raster images.
+  ///
+  /// For example, to blur an image using [Canvas.drawImage], apply an
+  /// [ImageFilter.blur]:
+  ///
+  /// ```dart
+  /// import 'dart:ui' as ui;
+  ///
+  /// ui.Image image;
+  ///
+  /// void paint(Canvas canvas, Size size) {
+  ///   canvas.drawImage(
+  ///     image,
+  ///     Offset.zero,
+  ///     Paint()..imageFilter = ui.ImageFilter.blur(sigmaX: .5, sigmaY: .5),
+  ///   );
+  /// }
+  /// ```
+  ///
+  /// See also:
+  ///
+  ///  * [MaskFilter], which is used for drawing geometry.
+  ImageFilter get imageFilter {
+    // TODO(dnfield): Implement ImageFilter
+    return null;
```

Comment:
Our approach is to file an issue for known missing Web features. We also typically throw an `UnimplementedError`, but if this would crash a test, then ignoring this value is fine.